### ISSUE TITLE
Link warning outdated

### ISSUE THREAD

**Issue by sneumann** [sneumann/mzR#25]
Reported by Emma Schymanski: 

Is the mzR warning message out of date? 
```
Warning message:
In fun(libname, pkgname) :
  mzR has been built against a different Rcpp version (0.11.5)
than is installed on your system (0.11.6). This might lead to errors
when loading mzR. If you encounter such issues, please send a report,
including the output of sessionInfo() to the Bioc mailing list at 
http://www.bioconductor.org/help/mailing-list. For details see also
https://github.com/sneumann/mzR/wiki/mzR-Rcpp-compiler-linker-issue.
```
-> this points people to the mailing list, whereas now they should visit the support site?
https://support.bioconductor.org/

**Comment by lgatto** [comment]
What version of Bioc, what platform?

**Comment by sneumann** [comment]
https://github.com/sneumann/mzR/blob/e5ccb9c8239746d66c3d0ee83c9ac7ce92c49d89/R/zzz.R.in#L9

Question is should we point people to bioc-devel or the new BioC support site ? I am not monitoring the latter.

**Comment by lgatto** [comment]
> Question is should we point people to bioc-devel or the new BioC support site ? I am not monitoring the latter.

I'll update to point to support site. I get emails when somebody posts a question about any package I maintain.

**Comment by lgatto** [comment]
> What version of Bioc, what platform?

I mean what platform/Bioc version is Emma working on? A quick look on the servers that biold Bioc 3.1 and 3.2 packages indicate that `Rcpp` 0.11.6 should be used.

**Comment by lgatto** [comment]
The warning will disappear at the next build.

**Comment by lgatto** [comment]
Also fixed in Bioc 3.1.
